microlensing_photometry/astrometry/wcs.py

```diff
@@ -0,0 +1,93 @@
+from skimage.registration import phase_cross_correlation
+import numpy as np
+from astropy.coordinates import SkyCoord
+import scipy.spatial as sspa
+from skimage.measure import ransac
+from skimage import transform as tf
+from astropy.wcs import WCS,utils
+
+import microlensing_photometry.logistics.GaiaTools.GaiaCatalog as GC
+
+def find_images_shifts(reference,image,image_fraction =0.25, upsample_factor=1):
+    """
+    Estimate the shifts between two images. Generally a good idea to do a fraction in the middle,
+    where the effect of rotation is minimal.
+
+    Parameters
+    ----------
+    reference : an image acting as the reference
+    image :  the image we want to align
+    image_fraction : the fraction of image around the center we want to analyze
+    upsample_factor : the degree of upsampling, if one wants subpixel accuracy
+
+
+    Returns
+    -------
+    shiftx : the shift in pixels in the x direction
+    shifty : the shift in pixels in the y direction
+    """
+
+    leny, lenx = (np.array(image.shape) * image_fraction).astype(int)
+    starty,startx = (np.array(image.shape)*0.5-[leny/2,lenx/2]).astype(int)
+
+    subref = reference.astype(float)[starty:starty+leny,startx:startx+lenx]
+    subimage = image.astype(float)[starty:starty+leny,startx:startx+lenx]
+    shifts, errors, phasediff = phase_cross_correlation(subref,subimage,
+                                                        normalization=None,
+                                                        upsample_factor=upsample_factor)
+
+    shifty,shiftx = shifts
+
+    return shiftx,shifty
+
+
+def refine_image_wcs(image, stars_image, image_wcs, gaia_catalog, star_limit = 1000):
+    """
+    Refine the WCS of an image with Gaia catalog
```

Comment:
As above, the comment needs to be more specific to be informative.  How is the WCS refined?
Similarly, types are needed for the parameters